### ISSUE TITLE
Broken with Foundry v9

### ISSUE THREAD

**Issue by mkahvi** [SvenWerlen/fvtt-data-toolbox#22]
For example
```
foundry.js:326 TypeError: Error thrown in hooked function '' for hook 'getItemSheetHeaderButtons': Cannot read properties of undefined (reading 'entity')
    at Function.hook (letscontribute.js:79)
    at letscontribute.js:67
    at Function._call (foundry.js:304)
    at Function.call (foundry.js:289)
    at ItemSheetPF._getHeaderButtons (foundry.js:3151)
    at ItemSheetPF._getHeaderButtons (foundry.js:4112)
    at ItemSheetPF._renderOuter (foundry.js:3036)
    at ItemSheetPF._render (foundry.js:2924)
    at async ItemSheetPF._render (foundry.js:3609)
```

**Comment by brunocalado** [comment]
Hi.

Please remove the foundry vtt warning.

Replace "compatibleCoreVersion":"0.9.0" by "compatibleCoreVersion":"9".

Docs: https://gitlab.com/foundrynet/foundryvtt/-/issues/5635

**Comment by mkahvi** [comment]
@brunocalado  You should post separate issue about that instead of commenting in completely unrelated one.

**Comment by SvenWerlen** [comment]
Hi @mkahvi , I'm so sorry but I had to stop supporting Let's contribute part of the module. 
The maintenance and support was to much compared to the number of people using it.
I'll leave the server up-and-running for a few months to let people finalize their contributions but will then plug it off.
Let me know how I can help you migrate to other alternatives

@brunocalado : I pushed a new version with V9 support
